docker/docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+version: '3.4'
+
+networks:
+  backend.fame:
+
+volumes:
+  mongodb:
+  modules:
+  storage:
+  avatars:
+
+services:
+  fame_web:
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile
+    image: fame:latest
+    hostname: fame_web
+    container_name: fame_web
+    networks:
+      - backend.fame
+    ports:
+      - "127.0.0.1:4200:4200"
+    env_file:
+      - fame.env
+    depends_on:
+      - fame_db
+    volumes:
+      - modules:/opt/fame/fame/modules/
+      - storage:/opt/fame/fame/storage/
+      - avatars:/opt/fame/web/static/img/avatars/
+
+  fame_worker:
+    build:
+      context: ../
+      dockerfile: ./docker/Dockerfile
+    image: fame:latest
+    hostname: fame_worker
+    container_name: fame_worker
+    entrypoint: /bin/sh
+    command: -c "wait-for -t 15 fame_web:4200 && /opt/fame/docker/docker-entrypoint.sh worker"
+    networks:
+      - backend.fame
+    env_file:
+      - fame.env
+    privileged: true # this is required to use docker-in-docker functionnalities. Without privilegied mode, FAME modules which are using docker will not work
+    depends_on:
+      - fame_db
+      - fame_web
+
+  fame_updater:
```

Comment:
I don't really get the point of having two workers in this deployment. I would expect only one local worker, but two is fine.

Reply:
Well, I just tried to respect the docker philosophy here 😄 

Ideally, each docker should be used for one task only. This push you to "break down" a monolithic application into multiple "microservices". 

That being said : if you want I can indeed move back the update function to the web server. Could you confirm you would like me to do that?